form.js

```diff
@@ -15,3 +19,33 @@ form.addEventListener('submit', (e) => {
     form.submit();
   }
 });
+
+let localData = JSON.parse(localStorage.getItem('userInput'));
+let formInput = {
+  name: document.getElementById('name').value,
+  email: document.getElementById('email').value,
+  message: document.getElementById('message').value,
+};
+
+const isInputChange = (input) => {
+  input.addEventListener('change', (e) => {
+    formInput = {
+      name: document.getElementById('name').value,
+      email: document.getElementById('email').value,
+      message: document.getElementById('message').value,
+    };
+    formInput[input.name] = e.target.value;
+    localStorage.setItem('userInput', JSON.stringify(formInput));
+    localData = JSON.parse(localStorage.getItem('userInput'));
+  });
+};
+
+isInputChange(nameInput);
+isInputChange(emailInput);
+isInputChange(messageInput);
+
+if (localData !== null) {
+  nameInput.value = localData.name;
+  emailInput.value = localData.email;
+  messageInput.value = localData.message;
+}
```

Comment:
Your data is not being saved in browser local storage. Kindly fix it. Also, there is a type error in your code. Kindly fix that also. 
![image](https://user-images.githubusercontent.com/33268927/173820040-e878c99b-b397-453d-a7db-9ee2546e448a.png)